src/photos.cs

```diff
@@ -1406,13 +1424,15 @@ interface PHProjectChangeRequest {
 	}
 
 	[Mac (10,13)]
-	[NoiOS][NoTV]
-	[Unavailable (PlatformName.MacCatalyst)]
+	[TV (15,0)]
+	[iOS (15,0)]
 	[Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
that needs to be removed as you already removed the `[Unavailable (PlatformName.MacCatalyst)]`

Reply:
The [Advice]? Doesn't that apply to macOS?

Reply:
"UIKit on macOS" => Mac Catalyst (it's an old name).

Reply:
ah, good to know! thanks!